packages/babel-plugin-transform-class-properties/src/index.js

```diff
@@ -85,9 +85,9 @@ export default function({ types: t }) {
           const propNode = prop.node;
           if (propNode.decorators && propNode.decorators.length > 0) continue;
 
-          // In non-spec mode, all properties without values are ignored.
+          // In loose mode, all properties without values are ignored.
           // In spec mode, *static* properties without values are still defined (see below).
```

Comment:
_think_ i covered these now. if you've got a chance to take another look, let me know if anything is missing!